src/common/libterminus/client.c

```diff
@@ -271,8 +271,8 @@ static void pty_client_data (struct flux_pty_client *c, flux_future_t *f)
         llog_error (c, "unpack: %s", error.text);
         return;
     }
-    if (write (STDIN_FILENO, data, len) < 0) {
-        llog_error (c, "data decode failed: %s", strerror (errno));
+    if (write (STDOUT_FILENO, data, len) < 0) {
+        llog_error (c, "write %zu bytes: %s", len, strerror (errno));
         return;
```

Comment:
i don't know this code that well and this change looks odd in that there isn't a change on the other side of this.  I suppose the other side is listening to all stdio and that internal to "data" it has encoded if this is stdin/out/err or whatever?

Reply:
> I'm not sure what you mean about the other side of this?

I was meaning the server side.

> This is a tty, there is no stdin stdout stderr, or I should say all of them point to the same file descriptor.

That's what I assumed, otherwise this wouldn't make sense :-)  Not sure if a comment is worthwhile.

Reply:
Oh, on the server side there's no possibility of redirection like in the client

Reply:
Yeah, maybe a comment would have been a good idea